.github/workflows/tests.yml

```diff
@@ -97,7 +108,7 @@ jobs:
 
     - name: Run tests
       run: |
-        if [ "${{ matrix.distro.name }}" = alpine ]; then
+        if [ "${{ matrix.distro.name }}" = alpine ] && ([ "${{ matrix.distro.tag }}" = 3.10 ] || [ "${{ matrix.distro.variant }}" = "-lts" ]); then
```

Comment:
Above all - a HUGE thank you for the test.

Please mention in the commit message why we need these extra checks or alternatively remove them.

Reply:
The linux-{vanilla,virt}-dev in alpine 3.10 and the linux-lts-dev in alpine 3.1[1234] do not contain a compiled sign-file tool. We should test the signing related code only on other alpine environments.
https://pkgs.alpinelinux.org/contents?file=*sign-file*&path=&name=linux-vanilla-dev&branch=v3.10
https://pkgs.alpinelinux.org/contents?file=*sign-file*&path=&name=linux-lts-dev
https://pkgs.alpinelinux.org/contents?file=*sign-file*&path=&name=linux-virt-dev